cuebot/src/main/java/com/imageworks/spcue/dispatcher/HostReportHandler.java

```diff
@@ -499,47 +500,88 @@ private void changeLockState(DispatchHost host, CoreDetail coreInfo) {
     }
 
     /**
-     * Prevent host from entering an OOM state where oom-killer might start killing important OS processes.
+     * Prevent host from entering an OOM state where oom-killer might start killing
+     * important OS processes. and frames start using SWAP memory
      * The kill logic will kick in one of the following conditions is met:
-     *   - Host has less than OOM_MEMORY_LEFT_THRESHOLD_PERCENT memory available
-     *   - A frame is taking more than OOM_FRAME_OVERBOARD_PERCENT of what it had reserved
-     * For frames that are using more than they had reserved but not above the threshold, negotiate expanding
-     * the reservations with other frames on the same host
-     * 
+     * - Host has less than oom_max_safe_used_physical_memory_threshold memory
+     * available and less than oom_max_safe_used_swap_memory_threshold swap
+     * available
+     * - A frame is taking more than OOM_FRAME_OVERBOARD_PERCENT of what it had
+     * reserved
+     * For frames that are using more than they had reserved but not above the
+     * threshold, negotiate expanding the reservations with other frames on the same
```

Comment:
negotiate `to expand` the ...

Reply:
I think present continuous makes more sense here.